unittests/test-contracts/test_api/test_compiler_builtins.cpp

```diff
@@ -0,0 +1,424 @@
+/**
+ *    @file test_compiler_builtins.cpp
+ *    @copyright defined in eos/LICENSE
+ */
+#include <eosio/eosio.hpp>
+#include <eosio/print.hpp>
+
+#include "test_api.hpp"
+
+//these are no longer exposed in cdt
```

Comment:
Similarly, we need to explicitly call the host functions so we know we're actually testing the host functions. It's possible future cdt versions won't ever implicitly call the host functions given something like
```c++
__int128 a = 0, b = 0;
a/b;
```